### PR TITLE
Checkout updates 2023

### PR DESCRIPTION
**Pull request by karenword**
Proposed updates to address changes to the Instructor checkout process. Not to be merged until rollout.

### CODE REVIEW

**Review by Talishask — CHANGES_REQUESTED** [review]
Shared some updates to provide a full overview of checkout at the top, with folks only needing to go into the bottom sections if they are looking for specific details.

File: learners/checkout.md [hunk 1]
```diff
@@ -8,283 +8,316 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
 
-After completing the Instructor Training workshop, there are three final steps to complete before qualifying as an Instructor.
+After attending Instructor Training, there are three steps to complete before qualifying as a Certified Carpentries Instructor.
 Briefly, the three steps are:
 
-1. [Make a small contribution to a lesson or glossary](#contributions).
-2. [Take part in an online community discussion session](#discussion).
-3. [Take part in an online teaching demonstration session](#demonstration).
+1. [Attend a Welcome Session](#welcome). A 1-hour onboarding to the Instructor and global Carpentries community.
+2. [Complete a Teaching Demonstration](#demonstration). A 1-hour session where trainees demonstrate 5 minutes of teaching and receive and share feedback with their peers.
```
Comment (karenword) [inline learners/checkout.md:15]:
```suggestion
2. [Complete a Teaching Demonstration](#demonstration). A 1-hour session where trainees demonstrate 5 minutes of participatory instruction and share feedback, with evaluation by a Carpentries Instructor Trainer.
```

File: learners/checkout.md [hunk 2]
```diff
@@ -8,283 +8,320 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
+3. [Get Involved!](#getinvolved). Choose one more step to further your engagement with The Carpentries.
 
+#### How will the Checkout steps be tracked?
+Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts. **For the “Get Involved” step, trainees must login to AMY to report which participation option they have completed.**
```
Comment (karenword) [inline learners/checkout.md:19]:
```suggestion
 **For the “Get Involved” step, trainees must login to AMY to report which participation option they would like to have counted.**  Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts.
```

File: learners/checkout.md [hunk 3]
```diff
@@ -8,283 +8,320 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
 All trainees have 3 months (90 days) from the end date of your training to complete checkout. If you need more time,
 3-month extensions may be requested by emailing [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org).
 Extensions may be granted for any reason up to 1 year from your training date.
 
-## 1\. Lesson Contributions {#contributions}
+:::::::::::::::::::::::::::::::::::::::::  callout
+## Help! I have a Question about Checkout! 
 
-### Keep It Simple!
+I have you have a question, you can email us at instructor.training@carpentries.org or visit the [#checkout-support](https://swc-slack-invite.herokuapp.com/) slack channel.
```
Comment (klbarnes20) [inline learners/checkout.md:29]:
```suggestion
If you have a question, you can email us at instructor.training@carpentries.org or visit the [#checkout-support](https://swc-slack-invite.herokuapp.com/) slack channel.
```

File: learners/checkout.md [hunk 4]
```diff
@@ -8,283 +8,320 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
-We are actively invested in making our processes as friendly and inclusive as possible, and that includes supporting our
-community in surmounting the barriers presented by our own technologies of choice. Watch our channels for
-opportunities to develop your skills as a contributor! In the mean time, if you
-have any ideas about how we can make our resources more contributor-friendly,
-please [let us know][contact-page].
+:::::::::::::::::::::::::::::::::::::::::  callout
 
+## Carpentries Terminology: Lessons vs Episodes
+A "lesson" corresponds to a single line in the lesson table and a single repository in GitHub
+(An example of a lesson: [R for Reproducible Scientific Analysis][r-gapminder]). The episodes within a lesson are numbered and listed in a collapsible window on the left side of the main lesson page. After 24 July, 2023, **you do NOT need to prepare a full lesson**, but should select a single episode within that lesson.
```
Comment (klbarnes20) [inline learners/checkout.md:118]:
```suggestion
(An example of a lesson: [R for Reproducible Scientific Analysis][r-gapminder]). The episodes within a lesson are numbered and listed in a collapsible window on the left side of the main lesson page. After 14 August, 2023, **you do NOT need to prepare a full lesson**, but should select a single episode within that lesson. 
```

**Review by klbarnes20 — APPROVED** [review]
Made two small changes, but otherwise it looks great!

File: learners/checkout.md [hunk 5]
```diff
@@ -8,283 +8,320 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
 
-### Community Discussions are Not Just for Checkout!
+Did you serve as a Helper or un-certified co-Instructor before you attended Instructor Training? 
+That counts! 
 
-Community Discussion meetings are an excellent opportunity to
-get help as you prepare to teach and to engage in vital reflection on your teaching practice while connecting with a community of like-
-minded Instructors. You are invited to attend these at any time! If you think you might enjoy hosting them, or would like to help develop
-other opportunities for Instructors, consider joining the Instructor Development Committee.
+### Attend an Instructor Meeting, Regional Call, or Other Community Programming
```
Comment (karenword) [inline learners/checkout.md:189]:
```suggestion
### Attend a Regional Call, Skill-Up Session, or Other Community Programming
```

File: learners/checkout.md [hunk 6]
```diff
@@ -8,283 +8,320 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
-workshop (we hear this a lot!) you will probably enjoy meeting people at your discussion session, too.
+## 3\. Get Involved! {#getinvolved}
 
-Being prepared to participate does not mean you need to do any homework in advance! However, it can help to put a little thought into
-what questions you have about training. In case you need a little inspiration, here are a few themes to consider:
+The purpose of this step is to encourage you to take a first step towards further engagement with 
+The Carpentries, choosing a path that works well for you. Any participation in Carpentries activities 
+or processes may be submitted to satisfy this requirement! We have a few suggestions to get you started.
+- Serve as an Instructor or a Helper at a workshop
+- Attend an Instructor Meeting, regional call, or other community programming
```
Comment (karenword) [inline learners/checkout.md:178]:
```suggestion
- Attend a regional call, skill-up session, or other community programming
```

File: learners/checkout.md [hunk 7]
```diff
@@ -8,283 +8,322 @@ discussioninfo: https://docs.carpentries.org/topic_folders/instructor_developmen
...
+The Carpentries offers a wide array of programming to meet the needs and interests of our community-members. Attending or hosting any of these 
+sessions can be used to fulfill your Get Involved step for Instructor checkout. For more on the different types of Carpentries community 
+programming, [see this blog post](https://carpentries.org/blog/2023/04/new-community-programming/) or go directly to the 
+[Community Sessions Etherpad][discussion] to look for sessions that will work for you!
 
-:::::::::::::::::::::::::::::::::::::::::  callout
+### Contribute to the construction or maintenance of Carpentries materials
+The Carpentries' lessons are built, sustained, and improved only through the collaborative effort of our community members. 
+All Instructors should feel empowered and invited to participate! Taking a first step to walk through the 
+contribution process is a great way to complete your Instructor certification.
```
Comment (karenword) [inline learners/checkout.md:200]:
```suggestion
contribution process is a great way to complete your Instructor certification.  
If you would like help making a lesson contribution consider making a post in the #first-time-contributors channel on [The Carpentries Slack][slack]. This channel is overseen by our Curriculum team and can be useful for contribution questions long after you have completed your checkout process. 
```

File: learners/checkout.md [hunk 8]
```diff
@@ -302,9 +342,9 @@ Approximately 1-2 weeks after your last step is complete, you should receive an
...
 [carpentries-incubator]: https://github.com/carpentries-incubator/
 [glosario-github]: https://github.com/carpentries/glosario/
-[discussion]: https://pad.carpentries.org/community-discussions
+[discussion]: https://pad.carpentries.org/community-sessions-2023
+[welcome]: https://pad.carpentries.org/welcome-sessions-2023
 [pad-of-pads]: https://pad.carpentries.org/pad-of-pads
-[mentoring]: https://docs.carpentries.org/topic_folders/instructor_development/mentoring_groups.html
 [demos-rubric]: https://carpentries.github.io/instructor-training/demos_rubric
 [r-gapminder]: https://swcarpentry.github.io/r-novice-gapminder/
 [start-points]: https://carpentries.github.io/instructor-training/demo_lessons
```
Comment (karenword) [inline learners/checkout.md:350]:
```suggestion
[start-points]: https://carpentries.github.io/instructor-training/demo_lessons
[slack]: https://swc-slack-invite.herokuapp.com/
```

**Comment by karenword** [comment]
Conflicts resolved and ready for merge on 13 August. Please check for conflicts and resolve promptly if any additional changes are made in the interim.

**Pull request merged**